plugins/ga.ts

```diff
@@ -9,7 +9,7 @@ export default function ga(options: GAOptions): Plugin {
   const main = `data:application/javascript,export default function(state){
       var ga = document.createElement("script");
       ga.type = "text/partytown";
-      ga.src = "/live/proxy/gtag/js?id=${trackingId}";
+      ga.src = "https://www.googletagmanager.com/gtag/js?id=${trackingId}";
```

Comment:
Just to clarify: this was only here because @guifromrio was using Brave and it had blocked GTM, right?

Reply:
Yeah, probably. I had this issue, then I disabled the brave `Shield UP` and worked fined, also at chrome it works normally